centreon/packages/js-config/cypress/e2e/commands.ts

```diff
@@ -399,6 +399,38 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add('insertDashboardWithPatch', (dashboardBody, patchBody) => {
```

Comment:
`insertDashboardWithPatch` is a weird name
Choose a more appropriate one like `insertDashboardDuringEditionOfExistingOne`